app/javascript/components/pages/ApartmentIndex.js

```diff
@@ -1,9 +1,20 @@
 import React, { Component } from 'react'
+import { Card, CardImg, CardText, CardDeck, CardBody,
+  CardTitle, CardSubtitle, Button, Col, Row } from 'reactstrap';
 
 export default class ApartmentIndex extends Component {
   render() {
+    const {
+      logged_in,
+      current_user
+    } = this.props
+    console.log("logged_in:", logged_in)
+    console.log("current_user:", current_user)
```

Comment:
Let's remove these console logs

Reply:
So this was an accident and I don't want to mess up Alex's hard work so I just copied what he had in main for this file and replaced what I had.